proxy/start_container_interceptor.go

```diff
@@ -11,7 +11,24 @@ import (
 
 type startContainerInterceptor struct{ proxy *Proxy }
 
+func callWeaveAndLog(container *docker.Container, description string, args ...string) error {
+	if _, stderr, err := callWeave(args...); err != nil {
+		return fmt.Errorf("Failed: %s container %s: %s", description, container.ID, string(stderr))
```

Comment:
Yeah, the previous behaviour was to log the message twice so I thought this would be an improvement.